generators/app/templates/src/components/Utils/Loader/loader.component.js

```diff
@@ -24,6 +27,7 @@ class Loader extends Component<Props> {
         <LoaderWrapper>
           <CubeGrid>
             {cubes.map((delay, i) => (
+              // eslint-disable-next-line react/no-array-index-key
```

Comment:
Is disable required? Is there a way to work around this?

Reply:
Fixed